src/components/ReviewList.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react"
+import ReviewCard from "./ReviewCard";
+
+export default function ReviewList () {
+
+const axios = require("axios");
+const [reviews, setReviews] = useState([]);
+
+useEffect(() => {
+    axios.get("https://nc-games-jk.herokuapp.com/api/reviews").then((response) => {
+        return response.data.reviews;
```

Comment:
Again, no more need for this .then block, you can have access to this straight away in the next then block!